test/presenters/bill-runs/index.bill-runs.presenter.test.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Thing under test
+const IndexBillRunsPresenter = require('../../../app/presenters/bill-runs/index-bill-runs.presenter.js')
+
+describe('Index Bill Runs presenter', () => {
+  let billRuns
+
+  describe('when provided with a populated bill run', () => {
+    beforeEach(() => {
+      billRuns = _billRuns()
+    })
+
+    it('correctly presents the data', () => {
+      const results = IndexBillRunsPresenter.go(billRuns)
+
+      expect(results).to.equal([
+        {
+          id: '31fec553-f2de-40cf-a8d7-a5fb65f5761b',
+          createdAt: '1 January 2024',
+          link: '/system/bill-runs/31fec553-f2de-40cf-a8d7-a5fb65f5761b',
+          number: 1002,
+          numberOfBills: 7,
+          region: 'Avalon',
+          scheme: 'sroc',
+          status: 'ready',
+          total: '£200.00',
+          type: 'Supplementary'
+        },
+        {
+          id: 'dfdde4c9-9a0e-440d-b297-7143903c6734',
+          createdAt: '1 October 2023',
+          link: '/system/bill-runs/dfdde4c9-9a0e-440d-b297-7143903c6734',
+          number: 1001,
+          numberOfBills: 15,
+          region: 'Albion',
+          scheme: 'sroc',
+          status: 'sent',
+          total: '£300.00',
+          type: 'Supplementary'
+        }
+      ])
+    })
+
+    describe("the 'link' property", () => {
```

Comment:
Could we add a test for the link property to test the link returned when the bill run has a status of review.

Reply:
🎯 💯